src/config.rs

```diff
@@ -50,7 +28,9 @@ pub struct Config {
 impl Default for Config {
     fn default() -> Self {
         Config {
-            kind: ConfigKind::default(),
+            defaults: Default::default(),
+            overrides: Default::default(),
+            sources: Default::default(),
             cache: Value::new(None, Table::new()),
```

Comment:
Just for consistency - is there a `Default` implementation of `Value`?

Reply:
Yes, but it actually does something different than this, IIRC, because the `Default` impl for `ValueKind` is actually `ValueKind::Nil` instead of an empty table.